.eslintrc.js

```diff
@@ -178,7 +178,6 @@ module.exports = {
         'no-throw-literal': 'error',
         'no-trailing-spaces': 'error',
         'no-undef-init': 'error',
-        'no-undefined': 'error',
```

Comment:
What a problem with that? I think is better to remove undefined vars

Reply:
It is not undefined vars but expression named 'undefined' and used to check if file is undefined

Reply:
Hm, that's ok